resources/windows/wix/sequences.wxs.erb

```diff
@@ -14,7 +14,9 @@
         CMDLINE_INSTALLDIR
       </Custom>
       <Custom Action='SetIniPropertyValues' After='AppSearch' />
-      <Custom Action='GetLocalisedAccountNames' After='AppSearch' />
+      <Custom Action='GetLocalisedAccountNames' After='AppSearch'>
```

Comment:
I completely misread what was going on here - I see that the condition is there only to prevent the action from running multiple times.  We probably don't need that bit? (or should leave a comment about what the condition is about)

Making sure `GetLocalisedAccountNames` appears in `InstallExecuteSequence` so it's always run in silent installation mode seems like the right thing to do.